src/pages/jihong/Login/Login.js

```diff
@@ -1,8 +1,102 @@
 import React from 'react';
-import './Login';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
+import '../../../../src/reset.scss';
+import { useState } from 'react';
 
 const Login = () => {
-  return <div></div>;
+  const [idInput, setIdInput] = useState('');
+  const [pwInput, setPwInput] = useState('');
+
+  const handleIdInput = event => {
+    setIdInput(event.target.value);
+  };
+
+  const handlePwInput = event => {
+    setPwInput(event.target.value);
+  };
+
+  const [isActive, setIsActive] = useState(false);
+
+  const isPassedLogin = () => {
+    return idInput.includes('@') && pwInput.length > 4
+      ? setIsActive(true)
+      : setIsActive(false);
+  };
```

Comment:
위와 연관지어서, 변수로 만들어서 사용
```suggestion
  const isLoginValid = idInput.includes('@') && pwInput.length > 4;
```